src/adapters/stickyadstv.js

```diff
@@ -0,0 +1,265 @@
+    "  preloadedVast:vast,"+
+    "  autoPlay:true"+
+    "};"+
+    "var ad = new window.top.com.stickyadstv.vpaid.Ad(document.getElementById(\"stickyadstv_prebid_target\"),config);"+
+    "ad.initAd("+size[0]+","+size[1]+",\"\",0,\"\",\"\");"+
+
+    "</script>";
+
+    return divHtml+script;
+  };
+
+  var formatIntextHTML = function(bid){
+    var placementCode = bid.placementCode;
+
+    var config = bid.params;
+
+    //default placement if no placement is set
+    if(!config.hasOwnProperty("domId") && !config.hasOwnProperty("auto") && !config.hasOwnProperty("p") && !config.hasOwnProperty("article")){
+      config.domId = placementCode;
+    }
+
+    var script = "<script type='text/javascript'>"+
+        
+    "var vast =  window.top.stickyadstv_cache[\""+placementCode+"\"];"+
+    "var config = {"+
+    "  preloadedVast:vast";
+
+    for(var key in config){
+      //dont' send format parameter
+      //neither zone nor vastUrlParams value as Vast is already loaded
+      if(config.hasOwnProperty(key) && key !== "format" && key !== "zone" && key !== "zoneId" && key !== "vastUrlParams"){
+
+        script += ","+key+":\""+config[key]+"\"";
+      }
+    }
+    script += "};"+
+    
+    "window.top.com.stickyadstv.intextroll.start(config);"+
+
+    "</script>";
+
+    return script;
+  };
+
+  var formatScreenRollHTML = function(bid){
+    var placementCode = bid.placementCode;
+
+    var config = bid.params;
+
+    var script = "<script type='text/javascript'>"+
+       
+    "var vast =  window.top.stickyadstv_cache[\""+placementCode+"\"];"+
+    "var config = {"+
+    "  preloadedVast:vast";
+
+    for(var key in config){
+      //dont' send format parameter
+      //neither zone nor vastUrlParams values as Vast is already loaded
+      if(config.hasOwnProperty(key) && key !== "format" && key !== "zone" && key !== "zoneId" && key !== "vastUrlParams"){
+
+        script += ","+key+":\""+config[key]+"\"";
+      }
+    }
+    script += "};"+
+    
+    "window.top.com.stickyadstv.screenroll.start(config);"+
+
+    "</script>";
+
+    return script;
+  };
+
+  function formatAdHTML(bid, size){
+
+    var integrationType = bid.params.format;
+
+    var html = "";
+    if(integrationType === "intext-roll"){
+      html = formatIntextHTML(bid);
+    }
+    else if(integrationType === "screen-roll"){
+      html = formatScreenRollHTML(bid);
+    }
+    else {
+      html = formatInBannerHTML(bid,size);
+    }
+    
+    return html;
+  }
+  
+
+  function extractPrice(vast){
+    var priceData = vast.getPricing();
+
+    if(!priceData) {
+      console.warn("StickyAdsTV: Bid pricing Can't be retreived. You may need to enable pricing on you're zone. Please get in touch with your sticky contact.");
+    }
+    
+    return priceData;
+  }
+
+  function formatBidObject(valid, priceData, html, width, height){
+    var bidObject;
+    if(valid && priceData) {
+      // valid bid response
+      bidObject = bidfactory.createBid(1);
```

Comment:
How? the documentation says to use  "var bidObject = bidfactory.createBid(1);"
see: http://prebid.org/dev-docs/bidder-adaptor.html